Makefile

```diff
@@ -8,7 +8,8 @@ BUILDSTR := ${VERSION} (\#${LAST_COMMIT} $(shell date -u +"%Y-%m-%dT%H:%M:%S%z")
 
 YARN ?= yarn
 GOPATH ?= $(HOME)/go
-STUFFBIN ?= $(GOPATH)/bin/stuffbin
+GOBIN ?= $(HOME)/gobin
```

Comment:
This is not correct. There is no `$HOME/gobin` directory in the standard installation. By default, it is indeed `$GOPATH/bin`.

Reply:
No worries if that is not default i will change it to `$HOME/go/bin`